StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/ElementsCustomer.swift

```diff
@@ -20,24 +20,31 @@ struct ElementsCustomer: Equatable, Hashable {
     public static func decoded(fromAPIResponse response: [AnyHashable: Any]?) -> ElementsCustomer? {
         // Required fields
         guard let response,
-              let paymentMethodsArray = response["payment_methods"] as? [[AnyHashable: Any]],
+              let paymentMethodsWithLinkDetailsArray = response["payment_methods_with_link_details"] as? [[AnyHashable: Any]],
```

Comment:
Usage of `payment_methods` is now discouraged, and `payment_methods_with_link_details` will always be populated.